src/partials/work-together.html

```diff
@@ -1 +1,62 @@
-<section class="section work-together"></section>
+<section class="work-together">
+    <h2 class="title-together title-headline">
+        <span class="black-text">let’s</span>
+        <span class="green-text">work</span> <br>
+        <span class="green-text">together</span>
+    </h2>
+    <div class="title-contacts-message">
+        <div class="contacts-together">
+            <p class="title-text">Contact me to transform your ideas into exceptional solutions.</p>
+            <div class="title-contacts-tabdes">
+                <div class="icon-and-text">
+                    <p class="title-contacts">

```

Comment:
`<p class="title-contacts">` It seems out of place. Just get rid of this extra wrapper